openml/runs/trace.py

```diff
@@ -9,7 +13,7 @@ class OpenMLRunTrace(object):
     Parameters
     ----------
     run_id : int
-        OpenML run id
+        OpenML run id.
```

Comment:
I don't care about the tailing dot, but why doesn't our PEP8 check make any noise? ;)